BParkingNano/plugins/ElectronMerger.cc

```diff
@@ -174,14 +179,19 @@ void ElectronMerger::produce(edm::StreamID, edm::Event &evt, edm::EventSetup con
    // same here Do we need evts without trg muon? now we skip them
    if (skipEle) continue;   
    
+
    //pf cleaning    
    bool clean_out = false;
-   for(const auto& pfele : *pf) {
+   for(const auto& pfele : *ele_out) {
```

Comment:
In this way you clean against all the electrons, including the previous low pt, probably a better solution is to have a vector of pointers of PF.

Reply:
I did not realise thanks. even if I would call duplicated lowPt if there are several within dR < 0.01 wrt each other....
will implement the comment